package.json

```diff
@@ -98,7 +98,10 @@
     "semver": "^6.3.0",
     "testem-failure-only-reporter": "^0.0.1",
     "toml": "^3.0.0",
-    "typescript": "3.7.5"
+    "typescript": "^4.0.2"
+  },
+  "resolutions": {
+    "typescript": "^4.0.2"
```

Comment:
We should not need a resolution, what forced you to add this?

Reply:
broccoli-typescript-compiler, iirc.

I did it preemptively during the RC period, because prereleases don't resolve correctly. More recently, I looked in the lockfile and saw that something was asking for ~3.9.something and kept it in.